### PR TITLE
fix(api): fix advice mapper incorrectly mapping 'from' column

### PR DESCRIPTION
**Pull request by emil-placheta**
## Describe your changes
Mapper incorrectly mapping 'from' column

Ticket: https://pins-ds.atlassian.net/browse/ASB-2294

## Type of change 🧩

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: packages/applications-service-api/src/utils/advice.mapper.js [hunk 1]
```diff
@@ -17,7 +17,8 @@ const mapBackOfficeAdviceToApi = (advice) => {
 };
 
 const mapCommonAdviceFieldsToApi = (advice) => {
-	const [firstName, ...lastName] = advice && advice.from ? advice.from.split(' ') : undefined;
+	const [firstName, ...lastName] = advice && advice.agent ? advice.agent.split(' ') : undefined;
```
Comment (Philip-Philippou1) [inline packages/applications-service-api/src/utils/advice.mapper.js:20]: Have we moved the problem to advice.agent ?  agent can be empty, user can create s51 advice with from being only name, does this work?

Looks like it should but want to make sure.

Comment (emil-placheta) [reply packages/applications-service-api/src/utils/advice.mapper.js]: Ah, I assumed that `agent` (where we get first and last name) was required and always available and only `from` (organisation) was optional, but you are right, it's _either_ first name + lastname _OR_ organisation thats required (or both as org can be optional).

This change will return empty strings if we can't get the name out of `agent` and FE will default to from (org). If somehow neither are present, FE shows `Anonymous`

**Review by Philip-Philippou1 — APPROVED** [review]

**Pull request merged**